Userland/Utilities/ping.cpp

```diff
@@ -30,6 +30,7 @@ static int min_ms;
 static int max_ms;
 static const char* host;
 static int payload_size = -1;
+static int optional_header_max_size = 40;
```

Comment:
Since this is a constant it is nice to give that in the type.
So something like:
`static constexpr int optional_header_max_size = 40;`
Additionally maybe add a link to somewhere where this 40 comes from (I have no idea where you would link but from wherever you figured out this constant is fine).